configurations/generic/credhub.yml

```diff
@@ -2,9 +2,9 @@
   path: /releases/-
```

Comment:
wouldn't it to be better to break backwards compatibility and use more generic parts? We don't need to use `credhub-user`. Actually, we can remove this file altogether.